packages/request-client.js/src/api/request.ts

```diff
@@ -656,6 +656,65 @@ export default class Request {
     return requestData;
   }
 
+  /**
+   * Declare a refund is received for the declarative payment network
```

Comment:
wrong comment? It's important to clearly describe this function, as its name could be confusing. 

Maybe rename to `addDeclarativeDelegate`?